src/ripple/protocol/impl/Feature.cpp

```diff
@@ -118,6 +118,7 @@ detail::supportedAmendments ()
         "MultiSignReserve",
         "fixTakerDryOfferRemoval",
         "fixMasterKeyAsRegularKey",
+        "MemoFee"
```

Comment:
Need to also add "MemoFee" to `featureNames` in Feature.h.

Reply:
Please add a trailing comma to this line as well.